recipes/pysam/meta.yaml

```diff
@@ -20,22 +20,24 @@ requirements:
   build:
     - {{ compiler('c') }}
   host:
-    - htslib
-    - samtools
-    - bcftools
+    - htslib 1.9
```

Comment:
We should increase the version here: https://github.com/bioconda/bioconda-utils/blob/master/bioconda_utils/bioconda_utils-conda_build_config.yaml#L12